teams/rust-analyzer.toml

```diff
@@ -1,16 +1,14 @@
-name = "wg-rls-2"
+name = "rust-analyzer"
 subteam-of = "compiler"
-kind = "working-group"
 
 [people]
 leads = ["Veykril"]
 members = ["matklad", "lnicola", "Veykril", "flodiebold"]
 alumni = ["jonas-schievink"]
 
 [website]
-name = "RLS 2.0 working group"
+name = "rust-analyzer team"
 description = "Experimenting with a new compiler architecture tailored for IDEs"
```

Comment:
Nit: maybe update the description too since we're renaming it?

Reply:
@Veykril Does this description still fit or shall we update it?

Reply:
Maybe "Rust IDE support and error-resilient compiler front-end" or something like? @matklad?

I think Veykril is less around these days, not sure if I can push to this branch.

Reply:
Updated it to lnicola's suggestion